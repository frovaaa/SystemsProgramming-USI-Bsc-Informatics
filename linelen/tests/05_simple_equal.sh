#!/bin/bash
exec "$1" 13 13 <<EOF
1
22
333
4444
55555
666666
7777777
88888888
999999999
1010101010
11111111111
121212121212
1313131313131
14141414141414
151515151515151
1616161616161616
17171717171717171
181818181818181818
1919191919191919191
20202020202020202020

.
..
..a
...b
....c
.....d
......e
.......f
........g
.........h
..........j
...........k
............l
.............m
..............o
...............p
................q
.................r
..................s
...................t
....................u
.....................v
......................w
.......................x
........................y
.........................z
EOF
