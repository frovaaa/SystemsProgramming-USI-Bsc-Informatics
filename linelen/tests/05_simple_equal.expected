1313131313131
............l
